test/adapters/dio_adapter_test.dart

```diff
@@ -38,8 +35,11 @@ void main() {
       test('Test that throws raises custom exception', () async {
         final dioError = DioError(
           error: {'message': 'error'},
-          response: Response(statusCode: 500),
-          type: DioErrorType.RESPONSE,
+          response: Response(
+            statusCode: 500,
+            request: RequestOptions(path: '/foo'),
```

Comment:
A bit of nitpicking but maybe passing constant declared at the beginning of the file - `path` will make more sense in this case just for the sake of consistency, though it essentially changes nothing.